streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -225,7 +480,73 @@ private void verifyCanGetByKey(final String[] keys,
         assertThat(countState, equalTo(expectedCount));
     }
 
-    private void waitUntilAtLeastOneRecordProcessed() throws InterruptedException {
+    /**
+     * Note: this method changes the values in expectedWindowState and expectedCount
+     * @param keys All the keys we ever expect to find
+     * @param expectedWindowState Expected window state
+     * @param expectedCount Expected cound
+     * @param windowStore Window Store
+     * @param myCount CountStore
+     * @param failIfKeyNotFound if true, tests fails if an expected key is not found in store. If false,
+     *                          the method merely inserts the new found key into the list of
+     *                          expected keys.
+     * @throws InterruptedException
+     */
+    private void verifyGreaterOrEqual(final String[] keys,
+                                      Map<String, Long> expectedWindowState,
+                                      Map<String, Long> expectedCount,
+                                      final ReadOnlyWindowStore<String, Long> windowStore,
```

Comment:
nit: could we rename to `expectedWindowedCount`, `expectedCount`, and `windowStore`, `keyValueStore`?